### ISSUE TITLE
Escaped HTML characters in output

### ISSUE THREAD

**Issue by emptyflask** [yosiat/panko_serializer#58]
Even with `Oj.mimic_JSON` set, there doesn't seem to be a way to output the non-escaped characters.
```json
{"other_serializer": {"body":"&nbsp;<p>..."}}
{"panko_serializer": {"body":"\u0026nbsp;\u003cp\u003e..."}}
```
It doesn't seem to be due to the Oj::StringWriter, so perhaps this is happening at a lower level:
```ruby
puts Oj::StringWriter.new.tap{|w| w.push_json '"<p>&nbsp;</p>"', "body"}.to_s
# "body":"<p>&nbsp;</p>"
```

**Comment by emptyflask** [comment]
I don't think this matters much in practice, it seems like most clients are able to handle it.

**Comment by yosiat** [comment]
Hi @emptyflask !

Sorry for not getting in touch about this PR!

Some questions before I can dig in, I assume you serialize ActiveRecord objects:

- Which version of Rails and Ruby are you using?
- Which DB provider? MySQL / PostgreSQL / SQLite ?
- What's the value stored in DB? `<p>&nbsp;</p>` ?

**Comment by yosiat** [comment]
@emptyflask hi! can you give any answers to the questions I asked? I think this is a quick one to fix, just need your input here so I can reproduce it correctly.

**Comment by emptyflask** [comment]
Oh, sorry, I got buried with other stuff and forgot to respond to this.

I will have to do some testing again, but I believe this was with Rails 5.x and PostgreSQL, and "&nbsp;" being the text in question stored in the db.

**Comment by emptyflask** [comment]
Ruby 2.6.3
Rails 6.0 (in this case I don't think it matters)

```ruby
class FooSerializer < Panko::Serializer
  attributes :body
end

FooSerializer.new.serialize(Struct.new(:body).new("<p>&nbsp;</p>")).to_json
# => {"body":"\u003cp\u003e\u0026nbsp;\u003c/p\u003e"}
```

**Comment by khaile** [comment]
Hi @yosiat, 
Could we consider setting the OJ mode to `:custom`?

I encountered the same issue and found that monkey patching the gem resolves it.

```ruby
module Panko
  class ArraySerializer
    def serialize_to_json(subjects)
      serialize_with_writer(subjects, Oj::StringWriter.new(mode: :custom)).to_s
    end
  end
end

module Panko
  class Serializer
    def serialize_to_json(object)
      serialize_with_writer(object, Oj::StringWriter.new(mode: :custom)).to_s
    end
  end
end
```
